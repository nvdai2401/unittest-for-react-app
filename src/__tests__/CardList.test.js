import React from 'react'

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import CardList from '../components/CardList'

it('expect to render CardList component', () => {
  const mockRobots = [
    {
      id: 2,
      name: 'Michael', 
      username: 'Mike',
      email: 'mike@example.com'
    }
  ]

  const wrapper = shallow(<CardList robots={mockRobots} />)
  expect(toJson(wrapper)).toMatchSnapshot()
})