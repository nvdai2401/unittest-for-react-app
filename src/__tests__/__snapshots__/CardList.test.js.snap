// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <Card
    email="mike@example.com"
    id={2}
    key="0"
    name="Michael"
  />
</div>
`;
